Code/Modules/player/ScenePlayer.h

```diff
@@ -296,7 +297,7 @@ static AP<ResourceType> loadResource(std::string fileName)
 }
 
 template <>
-static AP<mutant::anim_character_set> loadResource(std::string fileName)
+inline AP<mutant::anim_character_set> loadResource(std::string fileName)
```

Comment:
What is the motivation for using `inline` here?

I had issues with the `inline` keyword in other places (probably the C portion of the code?) while porting to another compiler.

My understanding is also that the `inline` keyword by itself is somewhat ambigious and unclear in C/C++ dialects ([See wikipedia](https://en.wikipedia.org/wiki/Inline_function#Storage_classes_of_inline_functions)).
I feel it should be removed or defined like `#define INLINE inline` so it's easier to fix when porting.

Reply:
the c++ standard has since changed to require that specialized templates require specific storage requirements and no longer allow static specialized templated functions.

Reply:
`inline` makes sense here, but that in turn breaks gcc410; so we need to handle this with a define or something